python/src/aiconfig/editor/client/src/LocalEditor.tsx

```diff
@@ -29,6 +26,31 @@ export default function Editor() {
     loadConfig();
   }, [loadConfig]);
 
+  const enableTelemetryIfAllowed = useCallback(async () => {
+    const res = await ufetch.post(ROUTE_TABLE.LOAD, {}); // change this to the endpoint used to get the config
+
+    const enableTelemetry = res.allow_usage_data_sharing;
+
+    if (enableTelemetry) {
+      datadogLogs.init({
+        clientToken: "pub356987caf022337989e492681d1944a8",
```

Comment:
What's this?

Reply:
datadog browser logging uses this token for authentication

Reply:
its generated in datadog (done in the datadog ui probably by ryan)